repo/packages/N/nexus/5/marathon.json.mustache

```diff
@@ -0,0 +1,70 @@
+{
+  "id": "{{service.name}}",
+  "cpus": {{service.cpus}},
+  "mem": {{service.mem}},
+  "acceptedResourceRoles": [ "{{service.resource-role}}" ],
+
+  "instances": 1,
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "{{resource.assets.container.docker.nexus3}}",
+      "network": "BRIDGE",
+      "portMappings": [
+        {
+          "containerPort": 8081, 
+          "hostPort": {{networking.host-port}} 
+        }
+      ]
+    },
+    "volumes": [
+      {{#storage.persist}}
+      {{^storage.external-volume.enable}}
+      {
+        "containerPath": "/nexus-data",
+        "hostPath": "{{storage.local-volume.host-volume}}",
+        "mode": "RW"
+      }
+      {{/storage.external-volume.enable}}
+      {{#storage.external-volume.enable}}
+      {
+        "containerPath": "/nexus-data",
+        "external": {
+          "name": "{{storage.external-volume.volume-name}}",
+          "provider": "dvdi",
+          "options": { "dvdi/driver": "rexray" }
+        },
+        "mode": "RW"
+      }
+      {{/storage.external-volume.enable}}
+      {{/storage.persist}}
+    ]
+  },
+  "healthChecks": [
+    {
+      "protocol": "COMMAND",
+      "command": { 
+        "value": "curl --user admin:admin123 --fail --request GET ${HOST}:${PORT0}/service/metrics/ping" 
```

Comment:
Just as a note: We would include these credentials in the example (I will update that anyhow).